server/routerlicious/packages/routerlicious/package.json

```diff
@@ -83,6 +83,8 @@
 		}
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {},
+		"disabled": true,
```

Comment:
This package has a bogus `main` and `types` entry: it has no actual `index.ts`. Typetests using the previous build setup happened to be OK with this, but flub is not. Thus I have disabled them since the package has no API surface anyway.

It seems like if we want to publish this package, we should probably be exposing a script under `bin` or something to run routerlicious, but going to consider that out of scope for this change...

Reply:
Agreed. Maybe a (type-tests) entrypoint for each lambda would be an alternative, but with this being just a reference implementation, I'm not sure even that is necessary.